packages/team-assignment/src/ducks/assignments/index.ts

```diff
@@ -215,6 +214,6 @@ export const getAssignmentsArrayByPlanId = () => {
       // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
       assignments = get(assignmentsMap, planId) ?? [];
     }
-    return assignments.filter((obj) => moment(obj.toDate) >= moment());
+    return assignments;
```

Comment:
@machariamuguku does this mean expired assignments(unassignments) also get to show on the web?

Reply:
@p-netm yes. But we need guidance from @dubdabasoduba

Reply:
there is no diff between revoked assignments and expired assignments both have a toDate that is in the past, Removing this check means once a user removes an assignment they will still see it on the web.
But I agree @dubdabasoduba should confirm what's the expected behavior here.

Reply:
@machariamuguku we should not display team assignments that have a `toDate` that is in the past compared to the date today.

Reply:
Updated @p-netm @dubdabasoduba